### ISSUE TITLE
Java Parses Exception when attempting to pass (int[], int) as method params

### ISSUE THREAD

**Issue by jonathan-m-phillips** [kelloggm/specimin#37]
## Issue:
Below are the commands provided, the class/method targeted, and the exception thrown. I am not positive if I am running Specimin incorrectly here, or if I am running into a bug.

### Command:
% ./gradlew run --args='--outputDirectory "/my/output/dir" --root "my/project/root/java" --t
argetFile "ArraySorter.java" --targetMethod "ArraySorter#getSortedArrayLargerThanIndex(int[], int)"'

### Class:
```
public class ArraySorter {

    public static int[] getSortedArrayLargerThanIndex(int[] array, int index) {
        if (index < 0 || index >= array.length) {
            return new int[0];
        }

        int count = 0;
        for (int i = 0; i < array.length; i++) {
            if (array[i] > array[index]) {
                count++;
            }
        }

        int[] resultArray = new int[count];

        int j = 0;
        for (int i = 0; i < array.length; i++) {
            if (array[i] > array[index]) {
                resultArray[j++] = array[i];
            }
        }

        Arrays.sort(resultArray);

        return resultArray;
    }
}
```

### Exception:
> Task :run FAILED
Exception in thread "main" java.lang.UnsupportedOperationException: ResolvedArrayType{PrimitiveTypeUsage{name='int'}} is not a Reference Type
        at com.github.javaparser.resolution.types.ResolvedType.asReferenceType(ResolvedType.java:142)
        at org.checkerframework.specimin.TargetMethodFinderVisitor.visit(TargetMethodFinderVisitor.java:199)
        at org.checkerframework.specimin.TargetMethodFinderVisitor.visit(TargetMethodFinderVisitor.java:32)
        at com.github.javaparser.ast.body.MethodDeclaration.accept(MethodDeclaration.java:98)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:1129)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.NodeList.accept(NodeList.java:277)
        at com.github.javaparser.ast.visitor.ModifierVisitor.modifyList(ModifierVisitor.java:1168)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:243)
        at org.checkerframework.specimin.TargetMethodFinderVisitor.visit(TargetMethodFinderVisitor.java:146)
        at org.checkerframework.specimin.TargetMethodFinderVisitor.visit(TargetMethodFinderVisitor.java:32)
        at com.github.javaparser.ast.body.ClassOrInterfaceDeclaration.accept(ClassOrInterfaceDeclaration.java:92)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:1129)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.NodeList.accept(NodeList.java:277)
        at com.github.javaparser.ast.visitor.ModifierVisitor.modifyList(ModifierVisitor.java:1168)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:306)
        at com.github.javaparser.ast.visitor.ModifierVisitor.visit(ModifierVisitor.java:51)
        at com.github.javaparser.ast.CompilationUnit.accept(CompilationUnit.java:127)
        at org.checkerframework.specimin.SpeciminRunner.performMinimization(SpeciminRunner.java:143)
        at org.checkerframework.specimin.SpeciminRunner.main(SpeciminRunner.java:61)

FAILURE: Build failed with an exception.

### Commands:
% ./gradlew run --args='--outputDirectory "/my/output/dir" --root "my/project/root/java" --t
argetFile "njit/ashe/SocketClient.java" --targetMethod "njit.ashe.SocketClient#testSocket(int[])"'

### Class:
```
public class SocketClient {

    public String testSocket(int[] array) {
        String serverIp = "127.0.0.1";
        int serverPort = 8888;
        String testGPT = null;
        try {
            Socket socket = new Socket(serverIp, serverPort);
            OutputStream outputStream = socket.getOutputStream();
            String message = "Hello world";
            outputStream.write(message.getBytes());
            outputStream.flush();

        } catch (IOException ex) {
            throw new RuntimeException(ex);
        }

        return testGPT;
    }
```

**Comment by LoiNguyenCS** [comment]
Thank you for your post. This is a bug from Specimin. I will fix it and let you know soon.

**Comment by LoiNguyenCS** [comment]
This issue is fixed by #39.
